### DISCUSSION TITLE
Alternative Desktop Entry Actions

### DISCUSSION THREAD

**Discussion by ricklamers** [davatorium/rofi#1608] [Ideas/Feature Requests]
It was already discussed here: https://github.com/davatorium/rofi/issues/836

But that patch link is dead and the issue was closed automatically. Are there any plans to incorporate this into `rofi` or is this being rejected?

Love `rofi` by the way! Thank you for creating it.

**Comment by ricklamers** [comment]
Perhaps @DaveDavenport could repost it and we could collaborate on a PR?

**Reply by DaveDavenport** [reply]
you might want to check the reason why the issue was closed more carefully.

**Reply by ricklamers** [reply]
Whoops!

**Reply by ricklamers** [reply]
Works like a charm :)
